source/_integrations/calendar.markdown

```diff
@@ -54,6 +62,79 @@ automation:
         message: >-
           Event {{ trigger.calendar_event.summary }} @
           {{ trigger.calendar_event.start }}
+  mode: parallel
+```
+{% endraw %}
+
+{% enddetails %}
+
+{% details "Example: Calendar Event Light Schedule " %}
+
+This example consists of:
+- For the calendar entity ` calendar.device_automation`
+- When event summary contains `Front Lights`
+- Turn on and off light named `light.front` when the event starts and ends
+- Only runs for one event at a time
+
+{% raw %}
+```yaml
+automation:
+  alias: Front Light Schedule
+  trigger:
+    - platform: calendar
+      event: start
+      entity_id: calendar.device_automation
+    - platform: calendar
+      event: end
+      entity_id: calendar.device_automation
+  condition:
+  - condition: template
+    value_template: "{{ 'Front Lights' in trigger.calendar_event.summary }}"
```

Comment:
This needs to be indented, see also:

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>